### PR TITLE
Enabled deployer to Hostinger

### PR DESCRIPTION
**Pull request by dhidalgX**
This workflow will enabled deployments into Hostinger.

### CODE REVIEW

**Review by jcvernaleo — DISMISSED** [review]
OK

File: .github/workflows/purefinance_deployment copy.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+name: Deploy Purefinance
+
+on:
+  workflow_dispatch:
+    push:
+        branches:
+        - main
```
Comment (gabmontes) [inline .github/workflows/purefinance_deployment copy.yaml:7]: This is not properly indented and the default branch is `master`.

```suggestion
  push:
    branches:
    - master
```

File: .github/workflows/purefinance_deployment copy.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '18'
```
Comment (gabmontes) [inline .github/workflows/purefinance_deployment copy.yaml:20]: Let's use the version defined in `.nvmrc`:

```suggestion
          node-version-file: .nvmrc
```

File: .github/workflows/purefinance_deployment copy.yaml
Comment (gabmontes) [inline .github/workflows/purefinance_deployment copy.yaml:1]: Let's remove " copy" from this file name.

**Review by gabmontes — CHANGES_REQUESTED** [review]

File: .github/workflows/purefinance_deployment copy.yaml
Comment (dhidalgX) [reply .github/workflows/purefinance_deployment copy.yaml:1]: File renamed

**Review by gabmontes — DISMISSED** [review]

**Comment by gabmontes** [comment]
@dhidalgX can you please sing all your commits?

**Comment by dhidalgX** [comment]
@gabmontes I can do a _squash and merge_ ,  is that what you are refering? The updates will be pushed as one.

**Pull request merged**